subprojects/model-core/src/main/java/org/gradle/internal/properties/annotations/NestedBeanAnnotationHandler.java

```diff
@@ -41,30 +41,55 @@ public boolean isPropertyRelevant() {
         return true;
     }
 
+    @Override
+    public void visitPropertyValue(String propertyName, PropertyValue value, PropertyMetadata propertyMetadata, PropertyVisitor visitor) {
+    }
```

Comment:
❓  Is this override necessary here?

Reply:
I believe so, `PropertyAnnotationHandler::visitPropertyValue` is not a default method and needs to be implemented, `ValidationProblemPropertyAnnotationHandler` has a similar override.

Reply:
I see this was not added here, but only moved a few lines higher.